auth/src/main/java/com/firebase/ui/auth/ui/phone/CheckPhoneNumberFragment.java

```diff
@@ -223,6 +192,53 @@ private void setCountryCode(PhoneNumber number) {
                 new Locale("", number.getCountryIso()), number.getCountryCode());
     }
 
+    private void setupCountrySpinner() {
+        Bundle params = getArguments().getBundle(ExtraConstants.PARAMS);
+        mCountryListSpinner.init(params);
+
+        setDefaultCountryForSpinner();
+
+        // Clear error when spinner is clicked on
+        mCountryListSpinner.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                mPhoneInputLayout.setError(null);
+            }
+        });
+    }
+
+    private void setDefaultCountryForSpinner() {
```

Comment:
This method needs to move back to where it was unfortunately (in `onActivityCreated`). Try clicking the phone number in your PR when you're at the submit code fragment. It will go back, but it will also show the phone picker again assuming you didn't pre-fill it in the AuthUI builder.

To see why that happens, read the comment where `mCalled` is used.